src/cljs/commiteth/ui_model.cljs

```diff
@@ -0,0 +1,121 @@
+                                  ::bounty-filter-type.predicate                     (fn [filter-value-interval bounty]
+                                                                                       (when-let [created-at-inst (:created-at bounty)]
+                                                                                         (let [created-at-date (-> created-at-inst inst-ms t-coerce/from-long)]
+                                                                                           (t/within? filter-value-interval created-at-date))))}
+   ::bounty-filter-type|owner    {::bounty-filter-type.name                                  "Owner"
+                                  ::bounty-filter-type.category                              ::bounty-filter-type-category|multiple-dynamic-options
+                                  ::bounty-filter-type.re-frame-subscription-key-for-options :commiteth.subscriptions/open-bounties-owners
+                                  ::bounty-filter-type.predicate                             (fn [filter-value bounty]
+                                                                                               (->> filter-value
+                                                                                                    (some #{(:repo-owner bounty)})
+                                                                                                    boolean))}})
+
+(def bounty-filter-types (keys bounty-filter-types-def))
+
+(defn bounty-filter-type->name [filter-type]
+  (-> bounty-filter-types-def (get filter-type) ::bounty-filter-type.name))
+
+(defn bounty-filter-type-date-option->name [option]
+  (bounty-filter-type-date-options-def option))
+
+(defn bounty-filter-value->short-text [filter-type filter-value]
+  (cond
+    (= filter-type ::bounty-filter-type|date)
+    (bounty-filter-type-date-option->name filter-value)
+
+    (#{::bounty-filter-type|owner
+       ::bounty-filter-type|currency} filter-type)
+    (str (bounty-filter-type->name filter-type) " (" (count filter-value) ")")
+
+    (= filter-type ::bounty-filter-type|value)
+    (str "$" (first filter-value) "-$" (second filter-value))
+
+    :else
+    (str filter-type " with val " filter-value)))
+
+(defn filter-bounties [filters-by-type bounties]
+  (let [filter-preds (->> filters-by-type
+                          (remove #(nil? (val %)))
+                          (map (fn [[filter-type filter-value]]
+                                 (let [filter-type-def    (bounty-filter-types-def filter-type)
+                                       pred               (::bounty-filter-type.predicate filter-type-def)
+                                       pre-pred-processor (::bounty-filter-type.pre-predicate-value-processor filter-type-def)
+                                       filter-value       (cond-> filter-value
+                                                                  pre-pred-processor (pre-pred-processor filter-value))]
```

Comment:
Should probably be `pre-pred-processor pre-pred-processor`, explicitly passing `filter-value` in the form part of `cond->` is not necessary here.

Reply:
Nice catch, fixed! :)